client/src/components/Player.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Player = props => {
+  return (
+    <div>
+      <h3>{props.player.name}</h3>
+      {console.log(props.player)}
+      <ul>
+        {props.player.hand.map((c, i)=> `${c.color} ${c.number}`)}
+      </ul>
+    </div>
+  )
+}
+
+module.exports = Player;
```

Comment:
we aren't using the build process from the original react curriculum. We should be using `export default Player;` but since this is a pure component we could just do `export default (props) => { }` on line 3